tests/locators_people_page.py

```diff
@@ -0,0 +1,40 @@
+from selenium.webdriver.common.by import By
+
+from config.TestData import TestData
+
+USER = 'j_username'
+PASSWD = 'j_password'
+SUBMIT = 'Submit'
+PEOPLE = (By.XPATH, '/html/body/div[4]/div[1]/div[1]/div[2]/span/a/span[2]')
+PEOPLE_LOGO = (By.XPATH, '/html/body/div[4]/div[2]/h1/img')
```

Comment:
Maybe PEOPLE and PEOPLE_LOGO should be under class PeoplePageLocator:?

Reply:
Of course!